extensions/interactions/codemirrorRequires.ts

```diff
@@ -16,12 +16,7 @@
  * @fileoverview Loads scripts needed for ui-codemirror.
  */
 
-const CodeMirror = require('static/code-mirror-5.17.0/lib/codemirror.js');
-Object.defineProperty(window, 'CodeMirror', {
-  value: CodeMirror,
-  writable: false
-});
-
+window.CodeMirror = require('static/code-mirror-5.17.0/lib/codemirror.js');
```

Comment:
The previous behavior is a but different am I right?

Reply:
Maybe. But the Merge View was working fine!